.github/workflows/ci.yml

```
@@ -0,0 +1,84 @@
+    runs-on: ubuntu-latest
+    environment: production
+
+    steps:
+    - name: Download build artifacts
+      uses: actions/download-artifact@v4
+      with:
+        name: build-artifacts
+        path: dist
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.4)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)